From 3ac0f3d10ce0ba4767bfb2a3ff30e608fb5853e1 Mon Sep 17 00:00:00 2001
From: Vladimir Oltean <olteanv@gmail.com>
Date: Wed, 26 Sep 2018 23:33:32 +0300
Subject: [PATCH 02/10] common: grant cnss-daemon access to sysfs_net

* It's scanning the /sys/class/net directory tree
* Also writing to /sys/class/net/wlan0/queues/rx-0/rps_cpus
* Yes, it's actually labeled as wcnss_service.

Change-Id: I2227868405d9ea6bfba1b9fa6103225b134c5759
Signed-off-by: Vladimir Oltean <olteanv@gmail.com>
---
 common/wcnss_service.te | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/common/wcnss_service.te b/common/wcnss_service.te
index a8863d4..9413af7 100644
--- a/common/wcnss_service.te
+++ b/common/wcnss_service.te
@@ -43,6 +43,8 @@ allow wcnss_service firmware_file:file r_file_perms;
 allow wcnss_service sysfs:file rw_file_perms;
 allow wcnss_service sysfs_data:file r_file_perms;
 allow wcnss_service proc:file r_file_perms;
+allow wcnss_service sysfs_net:file rw_file_perms;
+allow wcnss_service sysfs_net:dir r_dir_perms;
 
 # allow access to network performance tuner
 unix_socket_connect(wcnss_service, wigignpt, wigignpt)
-- 
2.17.1

